### ISSUE TITLE
Windows Support for LLVM

### ISSUE THREAD

**Issue by fniephaus** [oracle/graal#5007]
### TL;DR

We plan to extend the GraalVM LLVM runtime with support for Windows.

### Goals

As of GraalVM 22.2, the LLVM runtime is limited to Linux and macOS. We are adding support for Windows to the LLVM runtime, so that LLVM bitcode interpretation is possible on all platforms supported by GraalVM. While this may seem only interesting for languages that compile to LLVM, this change is also a prerequisite for other GraalVM languages such as the [Python](https://github.com/oracle/graalpython) and [Ruby](https://github.com/oracle/truffleruby/) runtimes to add support for Windows. The reason for this is that these runtimes build on the LLVM runtime to support, for example, C extensions.

### Non-Goals

- Add Windows support to GraalPy, TruffleRuby, and other GraalVM languages that build on the LLVM runtime (this may happen in the future)
- Backport Windows support to older releases of GraalVM

### Related Commits

- https://github.com/oracle/graal/commit/19117514fb3c3ba739894123e63a4063d90c775b, https://github.com/oracle/graal/commit/5bc3bfb2835979ca0eeac03ab26a5662b250b11e, https://github.com/oracle/graal/commit/f221ead428921886b9d0e1bb3cd90b33ab74411d, https://github.com/oracle/graal/commit/e8054afa7377559cb903c2bb99017370e369d7d2, and more

**Comment by fniephaus** [comment]
This has shipped with the 22.3 release. :tada:

**Comment by stdStudent** [comment]
@fniephaus, could you, please, explain how to use LLVM on Windows? I couldn't really find an instruction on how to use it...
Also, feel free to join this related discussion: https://github.com/oracle/graal/discussions/9776.
